views/sessions/login.erb

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Document</title>
+</head>
+<body>
+
+  <nav class="bg-gradient-to-r from-blue-300 to-blue-600 px-2 sm:px-4 py-2.5 rounded text-white">
+  <% if logged_in? %>
+    <div class="flex items-center justify-between">
+      <a href="/"><img src = "https://cdn-icons-png.flaticon.com/512/3338/3338265.png" alt =""/></a>
+      <!--Menu Items-->
+      <div class="flex space-x-6">
+        <a href="/">Home</a>
+        <a href="/users/<%= session[:user_id] %>">Profile</a>
+        <a href="/sessions/logout">Log Out</a>
+      </div>
+    </div>
```

Comment:
Maybe the user shouldn't be able to go on the login page if they are already logged in? Then you don't need to check if they are logged in in here.